plugins/focalfp/focalfp.quirk

```diff
@@ -0,0 +1,3 @@
+[HIDRAW\VEN_2808&DEV_0106]
```

Comment:
could this apply to all `VEN 2808`?

Reply:
I actually asked them to limit this for a first take; we don't know how well all the touchpads are going to respond to probing yet.